### DISCUSSION TITLE
The support server link on better discord is not working

### DISCUSSION THREAD

**Discussion by Tomask4** [community/community#37898] [Other Feature Feedback, Questions, & Ideas]
Hi qwert! Your support-server link on betterdiscord is not working... can you add new link?

**Comment by ossanna16** [comment]
Hi there @Tomask4 and welcome to our community! Can you please elaborate on your question?

**Reply by Tomask4** [reply]
the support-server link in better discord is not working: 
<img width="348" alt="Schermafbeelding 2022-11-02 om 08 42 46" src="https://user-images.githubusercontent.com/108363111/199429180-d289faac-b8f1-46e5-b87b-1870957af761.png">
<img width="465" alt="Schermafbeelding 2022-11-02 om 08 43 06" src="https://user-images.githubusercontent.com/108363111/199429195-81fdce78-8504-494d-8424-7973fdbf7587.png">
